t/43-cli-api.t

```diff
@@ -69,6 +69,9 @@ my @host = ('--host', $host);
 # Default options for authentication tests
 my @auth = ('--apikey', 'ARTHURKEY01', '--apisecret', 'EXCALIBUR', @host);
 
+# Job group template for template tests
+my $template_openqa = "$FindBin::Bin/data/job-templates/openqa.yaml";
```

Comment:
No reason for this to be outside the subtest.

Reply:
What do you mean? To test the changes to the `template=` parameter I need to pass a YaML configuration file. See L516.

Should I generate a file on the go from the subtest itself?

Reply:
> No reason for this to be outside the subtest.

Understood. Removed that, and I'm now using a tempfile created on the go.